src/visualization/parsers/layout.ts

```diff
@@ -193,6 +195,8 @@ export default class Layout implements IParser {
                     img.style.width = state.layout.width + "px";
                     img.style.height = state.layout.height + "px";
                 }
+                
+                
```

Comment:
revert this file

Reply:
not resolved - if you disagree mark as won'tfix and give an explanation

---
In reply to: [175853297](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175853297) [](ancestors = 175853297)